ibc-testkit/src/testapp/ibc/clients/mod.rs

```diff
@@ -61,6 +61,19 @@ impl From<ClientStateType> for AnyClientState {
     }
 }
 
+impl TryFrom<AnyClientState> for ClientStateType {
```

Comment:
I think this implementation is not needed anymore.

Reply:
[72ed7e9](https://github.com/cosmos/ibc-rs/pull/1127/commits/72ed7e996194209494a9ce8695644deb3f6cf0a2)